spec/app/ics-020-fungible-token-transfer/README.md

```diff
@@ -388,10 +413,9 @@ function onRecvPacket(packet: Packet) {
 
   if transferVersion == "ics20-1" {
      FungibleTokenPacketData data = UnmarshalJSON(packet.data)
-     trace, denom = parseICS20V1Denom(data.denom)
+     denom = parseICS20V1Denom(data.denom)
```

Comment:
THis is a bit confusing. Perhaps we can have better naming or a code comment on what each denom here represents

Reply:
I added a comment above to explain. Let me know if that's enough.